src/styles/custom.scss

```diff
@@ -1,14 +1,79 @@
+@import "~bootstrap/scss/bootstrap";
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,500;1,700&display=swap');
```

Comment:
We want to avoid expanding the scss files as much as possible. We're already importing the font here anyway:
https://github.com/Chicago-Artist-Guide/frontend-cag/blob/master/public/index.html#L27

Reply:
Removed Extra Import